tool/grind/utils.dart

```diff
@@ -3,43 +3,25 @@
 // https://opensource.org/licenses/MIT.
 
 import 'dart:async';
-import 'dart:convert';
 import 'dart:io';
 
-import 'package:archive/archive.dart';
+import 'package:cli_pkg/cli_pkg.dart' as pkg;
 import 'package:grinder/grinder.dart';
 import 'package:path/path.dart' as p;
-import 'package:pub_semver/pub_semver.dart';
-import 'package:yaml/yaml.dart';
-
-/// The version of the migrator.
-final String version =
-    loadYaml(File('pubspec.yaml').readAsStringSync())['version'] as String;
-
-/// The version of the current Dart executable.
-final Version dartVersion = Version.parse(Platform.version.split(" ").first);
 
 /// Options for [run] that tell Git to commit using SassBot's name and email.
 final sassBotEnvironment = RunOptions(environment: {
```

Comment:
Is this actually still used anywhere? Same question for a bunch of the other members in here.

Reply:
I originally just modified this to remove the same members that Dart Sass did, but, yeah, it looks like the only one that was actually still used was `environment`. I just moved it and the sanity check that used it into `tool/grind.dart` and deleted the `tool/grind` directory entirely.